ephios/core/templates/core/event_list.html

```diff
@@ -107,6 +108,9 @@ <h5 class="mb-0">
                                 <div class="col-auto col-lg-2 d-flex flex-row align-items-center justify-content-between">
                                     <p class="my-0 mr-1">
                                         <span class="fas fa-users"></span>
+                                        {% if event in events_user_can_change and events_user_can_change|has_requested_participations:event.pk %}
+                                            <span class="pending-indicator"></span>
+                                        {% endif %}
```

Comment:
Can't this be done using the stats below somehow? Still needs, the permission check, but maybe!?